examples/fargate-serverless/README.md

```diff
@@ -0,0 +1,66 @@
+# Serverless EKS Cluster using Fargate Profiles
+
+This example shows how to provision a serverless cluster (serverless data plane) using Fargate Profiles.
+
+There are some challenges to achieve this, primarily around the management and usage of CoreDNS on Fargate. There are alternative solutions shown on the internet, typically utilizing a `kubectl patch ...` command to update the compute type of the template spec annotation. This approach does work, but with the downside that the EKS service will eventually update the CoreDNS deployment and overwrite the patch, causing the CoreDNS deployment to fail due to insufficient capacity (it will be looking for EC2 compute when only Fargate is available).
```

Comment:
Should we move this example under `examples\node-groups`?

Reply:
Hmm, I'm not sure. Whats your thoughts/motivation around moving it to that directory?